src/Microsoft.AspNet.Mvc.Core/ModelBinding/Validation/DefaultCollectionValidationStrategy.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
+{
+    /// <summary>
+    /// The default implementation of <see cref="IValidationStrategy"/> for a collection.
+    /// </summary>
+    /// <remarks>
+    /// This implementation handles cases like:
+    /// <example>
+    ///     Model: IList&lt;Student&gt; 
+    ///     Query String: ?students[0].Age=8&students[1].Age=9
```

Comment:
Invalid `&` here. You caught some of these just below and later in the PR but should be `&amp;` in all new examples.